test/dlc_tests/conftest.py

```diff
@@ -711,18 +720,24 @@ def lookup_condition(lookup, image):
     """
     Return true if the ECR repo name ends with the lookup or lookup contains job type or device type part of the image uri.
     """
-    # Extract ecr repo name from the image and check if it exactly matches the lookup (fixture name)
-    repo_name = get_ecr_repo_name(image)
+    #Extract ecr repo name from the image and check if it exactly matches the lookup (fixture name)
+    repo_name = image.split("/")[-1].split(":")[0]
```

Comment:
why is this replaced by `get_ecr_repo_name`?

Reply:
it should maintain the same using `get_ecr_repo_name`